### PR TITLE
Update Vietnamese translation

### PR DESCRIPTION
**Pull request by phucbm**

### CODE REVIEW

File: packages/tools/locales/vi.po [hunk 1]
```diff
@@ -4147,223 +4159,216 @@ msgstr "Phiên bản thông tin: %s"
...
-#, fuzzy
-#~ msgid "Split"
-#~ msgstr "Chia đôi màn hình"
+msgid "Split"
+msgstr "Chia màn hình"
 
-#~ msgid "Resources"
-#~ msgstr "Tài nguyên"
+msgid "Resources"
+msgstr "Tài nguyên"
```
Comment (tessus) [inline packages/tools/locales/vi.po:4374]: How did these get uncommented? They should not even show up in Poedit.
These are most likely old strings.

File: packages/tools/locales/vi.po [hunk 2]
```diff
@@ -7,22 +7,26 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Joplin-CLI 1.0.0\n"
 "Report-Msgid-Bugs-To: \n"
-"Last-Translator: \n"
+"Last-Translator: PHUCBM\n"
```
Comment (tessus) [inline packages/tools/locales/vi.po:10]: Poedit would not add whitespace characters here.

Did you edit the file manually?

**Review by tessus — COMMENTED** [review]
Please see inline comments.

File: packages/tools/locales/vi.po (hunk 1 above)
Comment (phucbm) [reply packages/tools/locales/vi.po:4374]: I actually have no idea about these strings. I've just translated anything that shows up in Poedit.

File: packages/tools/locales/vi.po (hunk 2 above)
Comment (phucbm) [reply packages/tools/locales/vi.po:10]: You're right! I have manually edited the few first lines of the file while committing, that's my mistake didn't notice the whitespace.

File: packages/tools/locales/vi.po (hunk 1 above)
Comment (tessus) [reply packages/tools/locales/vi.po:4374]: Interesting. The file seems ok and these strings are not old translations.

File: packages/tools/locales/vi.po (hunk 2 above)
Comment (tessus) [reply packages/tools/locales/vi.po:10]: Can you open the file in Poedit and save it again? Would that remove the whitespace? If not, please fix the whitespace error.

Comment (phucbm) [reply packages/tools/locales/vi.po]: I've done as you suggested, the whitespace has been removed now.

**Comment by tessus** [comment]
Thanks for the update!

**Pull request merged**